docs/guide/tutorial-3-appearance.md

```diff
@@ -15,9 +15,16 @@ The result at the end of this tutorial is shown in [this figure](#simulation-aft
 The lighting of a world is determined by [Light](../reference/light.md) nodes and the [Background](../reference/background.md) .
 There are three types of light nodes: the [DirectionalLight](../reference/directionallight.md), the [PointLight](../reference/pointlight.md) and the [SpotLight](../reference/spotlight.md).
 A [DirectionalLight](../reference/directionallight.md) simulates a light which is infinitely far (ex: the sun), a [PointLight](../reference/pointlight.md) simulates light emitted from a single point (ex: a light bulb), and a [SpotLight](../reference/spotlight.md) simulates a conical light (ex: a flashlight).
+[This figure](#comparison-of-the-three-lighting-possible-in-webots-left-directionallight-middle-spotlight-and-right-pointlight) shows a comparison between them.
 Each type of light node can cast shadows.
 You can find their complete documentation in the [Reference Manual](../reference/nodes-and-api-functions.md).
 
+%figure "Comparison of the three lighting possible in Webots. Left DirectionalLight, middle SpotLight and right PointLight."
+
+![tutorial_light_comparison.png](images/tutorial_light_comparison.png)
+
+%end
```

Comment:
Very nice, this figure is indeed way better to understand the difference. But to make it clearer I would have enabled the shadows.